### DISCUSSION TITLE
How to disable animation behavior

### DISCUSSION THREAD

**Discussion by dinhlongviolin1** [LazyVim/LazyVim#5150] [Q&A] [answered]
Hi,
I was off for 2 weeks and when I am back, I upgraded lazyvim as usual and there was some delayed animation that I don't really want when moving up and down the page using C-U, C-D. It seems very similar to mini-animate to me but I don't see it within my lazyvim plugins. I have attached the behavior in a video above

https://github.com/user-attachments/assets/dcc953fb-65b6-4d8b-8b9d-46c0697e2404

**Comment by dpetka2001** [comment, marked as answer]
`vim.g.snacks_animate = false` in your `options.lua` file.

**Reply by dinhlongviolin1** [reply]
You are a life saver! Thanks for responding so quickly.

**Reply by mischavandenburg** [reply]
Thank you!

**Comment by EinGlasVollKakao** [comment]
There is also the option to just disable the scroll animation using `vim.g.snacks_animate_scroll = false` - useful if you like the other ones.

**Reply by dpetka2001** [reply]
And there's an option for `indent` specifically as well. I just assumed OP didn't want any animation behavior, in which case if that is not true then he should indeed do what you suggested.

**Reply by dinhlongviolin1** [reply]
Yes you are right. In my case I dont want any animation at all. But I will keep the other option as a comment so I can play with it later on. Thank you both for helping me!
